client/src/components/CommonComponents/LoaderProgress.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Backdrop, CircularProgress } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyle = makeStyles((theme) => ({
```

Comment:
split style into another file!

Reply:
i fixed it on the new branch for history front end